lisa/runners/lisa_runner.py

```diff
@@ -702,6 +702,11 @@ def _merge_test_requirements(
             is_allow_set=True, items=[platform_type]
         )
 
+        ignored_capability: List[schema.FeatureSettings] = []
```

Comment:
It doesn't need to convert them to feature settings, I think it just needs to compare feature's name with the ignored strings.

Reply:
updated